src/uwtools/resources/jsonschema/mpas-init.jsonschema

```diff
@@ -70,19 +70,7 @@
           "type": "string"
         },
         "streams": {
-          "properties": {
-            "path": {
-              "type": "string"
-            },
-            "values": {
-              "type": "object"
-            }
-          },
-          "required": [
-            "path",
-            "values"
-          ],
-          "type": "object"
+          "$ref": "urn:uwtools:mpas-streams"
```

Comment:
`mpas-init.jsonschema` and `mpas.jsonschema` now refer to a common sub-schema for the `streams:` block.